content/tutorials/02-embedded-programming.md

```diff
@@ -60,7 +60,7 @@ The RAM will contain garbage after startup. Our variables aren't initialized and
 
 To do this, we first have to tell the compiler to store our pre-initialized data somewhere in flash memory, so we can copy it into RAM at startup. This is done via the linker script:
 
-```ld
+```cpp
```

Comment:
You can check the commit message for more details on this, but essentially ld isn't a valid option for zine and causes the build to fail.